src/Libraries/RevitNodes/Elements/FailureMessage.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Autodesk.Revit.DB;
+using Autodesk.DesignScript.Runtime;
+using System;
+using RevitServices.Transactions;
+using RevitServices.Persistence;
+using System.Collections.Generic;
+using System.Collections;
+
+
+namespace Revit.Elements
+{
+    /// <summary>
+    /// Performance Adviser Failure Message
```

Comment:
Class name indicating it's for general purpose but comments seem to indicate it's only used for performance adviser, can we align these two?